app/Models/Meeting.php

```diff
@@ -2,13 +2,21 @@
 
 namespace App\Models;
 
-use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Carbon\Carbon;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\SoftDeletes;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
 
 class Meeting extends Model
 {
     use HasFactory, SoftDeletes;
+    const STATUS = [
+        'stand by'      => ['id' => 1, 'color' => 'warning'],
+        'in session'    => ['id' => 2, 'color' => 'success'],
```

Comment:
for array keys don't use `space` in it. it should be replaced by `underscore` (`_`) like `in session`, it should be `in_session`

Reply:
I changed it.